apps/web/pages/[chain].tsx

```diff
@@ -110,9 +119,20 @@ export default function Home({ isOutOfCELO }: Props) {
   )
 }
 
-export async function getServerSideProps(): Promise<{ props: Props }> {
-  const isOutOfCELO = await isBalanceBelowPar()
+export default Home
+
+export const getServerSideProps: GetServerSideProps<Props> = async (
+  context
+) => {
+  const network = context.query.chain
```

Comment:
sometimes querys can be Arrays (according to types)

Reply:
Look at the line below this